.travis.yml

```diff
@@ -34,8 +50,8 @@ before_install:
 script:
   - mkdir build
   - cd build
-  - cmake -DMATERIALX_BUILD_PYTHON=ON -DMATERIALX_INSTALL_PYTHON=OFF -DMATERIALX_WARNINGS_AS_ERRORS=ON ..
-  - cmake --build . --target install -- -j4
-  - ctest --output-on-failure
+  - cmake -DMATERIALX_BUILD_PYTHON=ON -DMATERIALX_INSTALL_PYTHON=OFF -DMATERIALX_WARNINGS_AS_ERRORS=ON -DCMAKE_BUILD_TYPE="${CONFIG}" ..
+  - cmake --build . --target install --config "${CONFIG}" -- -j4
+  - ctest --output-on-failure --build-config "${CONFIG}"
   - export PYTHONPATH=$PYTHONPATH:$PWD/installed/python
   - cmake -E chdir ../python/MaterialXTest python main.py
```

Comment:
Need to merge deployment in here as well.

Reply:
Same comment as above. When adding deploy you'll need a debug/relwithdebug string as part of the zip name.